distribution/carbon-home/conf/deployment.yaml

```diff
@@ -35,4 +33,4 @@ wso2.securevault:
   masterKeyReader:
     type: org.wso2.carbon.secvault.reader.DefaultMasterKeyReader
     parameters:
-      masterKeyReaderFile: ../../conf/${sys:wso2.runtime}/master-keys.yaml
+      masterKeyReaderFile: ../../conf/${sys:wso2.runtime}/master-keys.yaml
```

Comment:
deployment.yaml is specific for each product. Therefore IMO we need to either install this via a separate product specific feature or copy during distribution build using bin.xml.